src/ergw_sx_node_reg.erl

```diff
@@ -86,12 +86,16 @@ handle_death(_Pid, _Reason, State) ->
 handle_call({up, Key, Caps}, _From, State) ->
     case lookup(Key) of
 	{ok, Pid} ->
+	    ergw_prometheus:pfcp_sx_node_association(inc, Key, up),
+	    ergw_prometheus:pfcp_sx_node_association(dec, Key, down),
```

Comment:
whats the purpose of this inc-dec dance?

Reply:
The main reason of this dancing in calculation of total nodes what was `up` and `down`. If we will do only inc of `up` nodes this will be not correct because the same node can `down` at the moment of receiving the metrics, so we need to do inc of `up` metric if node will `down`. If we will need calculate all nodes what was run and try get only `up` nodes this will be also not correct because some f these nodes can be `down` and correct calculation will be `up + down = total`. But if this is not needed - the logic can be removed. Basically this implementation try separate these two different states.